app/Http/Controllers/PatientController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\PatientRequest;
+use App\Http\Requests\UpdatePatientRequest;
+use App\Models\Patient;
+use App\Http\Resources\PatientResource;
+
+class PatientController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $patient = Patient::all();
```

Comment:
Use laravel api resource for patients here.
https://laravel.com/docs/8.x/eloquent-resources